### DISCUSSION TITLE
Qgutter with Qselect

### DISCUSSION THREAD

**Discussion by unknown1337** [quasarframework/quasar#15590] [General - Components / Directives / etc] [answered]
How can I combine q-gutter-xx with a q-select component?

Goal: ensure that the q-input behaves the same as a q-btn
Observed: Q-intput always on separate line (not inline)
example: https://codepen.io/srtrst/pen/rNZZmpw?editors=101 

Thanks!

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/poOOwEd?editors=1000

**Reply by unknown1337** [reply]
THANKS
